plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java

```diff
@@ -188,8 +188,7 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 			throw new CoreException(new Status(IStatus.ERROR, CDKCoreActivator.PLUGIN_ID, "Call to minishift up has failed."));
 		}
 
-		IProcess process = addProcessToLaunch(p, launch,s);
-		linkTerminal(p);
```

Comment:
what I dont understand here is that "linkTerminal(process)" is gone now but in the code above you still to seem to support linking to the terminal or having the output in the console (see the switches based on "terminal"). Wrong?